tools/nni_cmd/launcher.py

```diff
@@ -344,13 +344,18 @@ def launch_experiment(args, experiment_config, mode, config_file_name, experimen
     '''follow steps to start rest server and start experiment'''
     nni_config = Config(config_file_name)
     # check packages for tuner
+    package_name, module_name = None, None
     if experiment_config.get('tuner') and experiment_config['tuner'].get('builtinTunerName'):
-        tuner_name = experiment_config['tuner']['builtinTunerName']
-        module_name = ModuleName[tuner_name]
+        package_name = experiment_config['tuner']['builtinTunerName']
+        module_name = ModuleName.get(package_name)
+    elif experiment_config.get('advisor') and experiment_config['advisor'].get('builtinAdvisorName'):
+        package_name = experiment_config['advisor']['builtinAdvisorName']
+        module_name = AdvisorModuleName.get(package_name)
+    if package_name and module_name:
         try:
-            check_call([sys.executable, '-c', 'import %s'%(module_name)])
-        except ModuleNotFoundError as e:
-            print_error('The tuner %s should be installed through nnictl'%(tuner_name))
+            exec('import %s' % module_name)
+        except ImportError:
```

Comment:
Why remove the ModuleNotFoundError ?

Reply:
ModuleNotFoundError  could not catch the error thrown by `check_call([sys.executable, '-c', 'import %s'%(module_name)])`